ffi/diplomat/ffi_coverage/src/main.rs

```diff
@@ -257,6 +275,32 @@ lazy_static::lazy_static! {
         // associated type
         "icu::plurals::PluralOperands::Err",
 
+        // Properties Rust internals
+        "icu::properties::maps::CodePointMapData::as_borrowed",
+        "icu::properties::maps::CodePointMapData::as_code_point_trie",
+        "icu::properties::maps::CodePointMapData::from_code_point_trie",
+        "icu::properties::maps::CodePointMapData::from_data",
+        "icu::properties::maps::CodePointMapData::to_code_point_trie",
+        "icu::properties::maps::CodePointMapData::try_into_converted",
+        "icu::properties::sets::CodePointSetData::as_borrowed",
+        "icu::properties::sets::CodePointSetData::as_code_point_inversion_list",
+        "icu::properties::sets::CodePointSetData::from_code_point_inversion_list",
```

Comment:
Issue: If we have `CodePointInversionList` available over FFI (which it seems like we should), these ones are not going to be "Rust internals"

Reply:
I'm not sure why we should?

Either way, at the moment, they're specific backend types.

Reply:
I assume these aren't scope for 1.0 anyway, so I can bump these over to the other section regardless.